scripts/start-sparkswapd.sh

```diff
@@ -7,7 +7,16 @@
 #            as `/secure` is a volume inside of the container
 ################################################
 
-set -eu
+set -u
+
+HOST_DOMAIN="host.docker.internal"
+echo $HOST_DOMAIN
```

Comment:
we can remove this echo and the `inside if` below. Also would make a note of why we need to include this inside of the file (because host.docker.internal is used for development networking and `host.docker.internal` doesn't work on linux)